README.md

```diff
@@ -1,350 +1,61 @@
-# Beneficiary FHIR Data Server (BFD)
+Beneficiary FHIR Data Server (BFD)
```

Comment:
I don't feel _too_ strongly about this, but... this GitHub Project and Repository covers more than _just_ the BFD Server. What do you think?
```suggestion
Beneficiary FHIR Data System (BFD)
```

Reply:
Yeah that makes sense. The original intent of the while project is to support the API server, but the repo is certainly the conglomeration of all the parts to get that running, so it makes sense to call it a system.